### PR TITLE
fix cpython in windows

### PR DESCRIPTION
**Pull request by yajiedesign**

### CODE REVIEW

File: python/mxnet/cython/ndarray.pyx [hunk 1]
```diff
@@ -13,7 +13,7 @@ cdef class NDArrayBase:
     cdef int cwritable
 
     cdef _set_handle(self, handle):
-        cdef unsigned long ptr
+        cdef unsigned long long ptr
```
Comment (piiswrong) [inline python/mxnet/cython/ndarray.pyx]: why long long? Does this break linux build? @tqchen

Comment (yajiedesign) [inline python/mxnet/cython/ndarray.pyx]: msvc long is 32bit.long long is 64bit.
in msvc,gcc,clang, long long all is 64bit.

Comment (piiswrong) [inline python/mxnet/cython/ndarray.pyx]: @tqchen Shouldn't this be void*? Is pointer type guaranteed to be 64bit?

Comment (tqchen) [inline python/mxnet/cython/ndarray.pyx]: Yes, pointer is gaunranteed to be 64 bit

Comment (tqchen) [inline python/mxnet/cython/ndarray.pyx]: The type is used to facilitate conversion in ctypes. the handle is stored in void*

Comment (piiswrong) [inline python/mxnet/cython/ndarray.pyx]: This wouldn't work on 32bit device right? Are there still any of those out there?

Comment (tqchen) [inline python/mxnet/cython/ndarray.pyx]: This will work on 32 bit device.

**Pull request merged**